Dockerfile

```diff
@@ -1,37 +1,61 @@
-FROM debian:7.8
+FROM ubuntu:xenial
 MAINTAINER Mike Babineau michael.babineau@gmail.com
 
 ENV \
     ZK_RELEASE="http://www.apache.org/dist/zookeeper/zookeeper-3.4.6/zookeeper-3.4.6.tar.gz" \
     EXHIBITOR_POM="https://raw.githubusercontent.com/Netflix/exhibitor/d911a16d704bbe790d84bbacc655ef050c1f5806/exhibitor-standalone/src/main/resources/buildscripts/standalone/maven/pom.xml" \
+    JVMTOP_RELEASE="https://github.com/patric-r/jvmtop/releases/download/0.8.0/jvmtop-0.8.0.tar.gz" \
     # Append "+" to ensure the package doesn't get purged
-    BUILD_DEPS="curl maven openjdk-7-jdk+" \
-    DEBIAN_FRONTEND="noninteractive"
+    BUILD_DEPS="oracle-java8-installer oracle-java8-set-default maven" \
+    DEBIAN_FRONTEND=noninteractive
+
+# Workaround for bug: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=807948
+RUN chmod 0777 /tmp
 
 # Use one step so we can remove intermediate dependencies and minimize size
-RUN \
+RUN set -ex \
     # Install dependencies
-    apt-get update \
-    && apt-get install -y --allow-unauthenticated --no-install-recommends $BUILD_DEPS \
+    && apt-get update -q \
+    && apt-get upgrade -yqq \
+    && apt-get install -y --no-install-recommends \
+    		tcl \
+    		tk \
+            wget \
+            curl \
+            ca-certificates \
+            procps \
+    && apt-get install -y --force-yes software-properties-common python-software-properties \
+    && /bin/echo oracle-java8-installer shared/accepted-oracle-license-v1-1 select true | debconf-set-selections \
+    && add-apt-repository -y ppa:webupd8team/java \
+    && apt-get update \
+    && apt-get install -y $BUILD_DEPS \
+    # Update java settings so DNS changes take hold.
+    && grep '^networkaddress.cache.ttl=' /etc/java-8-oracle/security/java.security || echo 'networkaddress.cache.ttl=60' >> /etc/java-8-oracle/security/java.security \
 
-    # Default DNS cache TTL is -1. DNS records, like, change, man.
-    && grep '^networkaddress.cache.ttl=' /etc/java-7-openjdk/security/java.security || echo 'networkaddress.cache.ttl=60' >> /etc/java-7-openjdk/security/java.security \
+    # Install jvmtop
+    && wget -O /tmp/jvmtop.gz "${JVMTOP_RELEASE}" \
+    && mkdir -p /opt/jvmtop \
+    && tar -xvzf /tmp/jvmtop.gz -C /opt/jvmtop \
+    && chmod +x /opt/jvmtop/jvmtop.sh \
+    && rm /tmp/jvmtop.gz \
 
     # Install ZK
-    && curl -Lo /tmp/zookeeper.tgz $ZK_RELEASE \
+    && wget -O /tmp/zookeeper.tgz "${ZK_RELEASE}" \
```

Comment:
Good question, for some reason, this curl -Lo was not working for me, my guess is that the version of curl that use to work fine on debian:jessie, doesn't work on xenial ... maybe because they're several versions different between each other, maybe because of how the flags used to compile it in general are different? not 100% sure, so I got fed up w/ it and used wget instead :) @kangman